src/Std/Data/DTreeMap/Lemmas.lean

```diff
@@ -3128,6 +3128,12 @@ theorem compare_minKey_modify_eq [TransCmp cmp] {k f he} :
       (t.minKey <| cast (congrArg (· = false) isEmpty_modify) he) = .eq :=
   Impl.Const.compare_minKey_modify_eq t.wf
 
+@[simp]
+theorem Ord.compare_minKey_modify_eq [Ord α] [TransOrd α] {t : DTreeMap α β} {k f he} :
```

Comment:
I'm not sure I like the `Ord` component in the name here. I think I would prefer `ordCompare_minKey_modify_eq`.

Reply:
Okay, that's fine for me, too.